addons/account/models/chart_template.py

```diff
@@ -644,6 +644,10 @@ def _post_load_data(self, template_code, company, template_data):
             company.account_purchase_tax_id = self.env['account.tax'].search([
                 *self.env['account.tax']._check_company_domain(company),
                 ('type_tax_use', 'in', ('purchase', 'all'))], limit=1).id
+        # Add tax_id to purchase default reconciliation model
+        reconcile_bill_model = self.ref("reconcile_bill", raise_if_not_found=False)
+        if reconcile_bill_model:
+            reconcile_bill_model.line_ids.tax_ids = [Command.set(company.account_purchase_tax_id.ids)]
```

Comment:
The `account_purchase_tax_id` is set after the compute method is triggered for the tax ids, so even though the compute is triggered, it still doesn’t prefill the tax for the Create Bill model. Perhaps this reconcile model could be created in the account demo, and in that case `account_purchase_tax_id` is already set in the company... but I'm not sure if it would be  a good idea to have it as a demo reco model. Maybe @william-andre you can see another solution so the tax for reconcile_bill doesn't have to be set here?

Reply:
One solution would be to have the compute method, and call it (or use `add_to_compute`) here

Reply:
Or you could also put a dependency on `company_id.account_purchase_tax_id`

Reply:
Or call `add_to_compute` from an inverse function on `<res.company>.account_purchase_tax_id` (maybe with something like `_conditional_add_to_compute` :magic_wand:)